src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/types.ts

```diff
@@ -93,3 +93,38 @@ export type ThorchainSaversDepositQuoteResponse =
 export type ThorchainSaversWithdrawQuoteResponse =
   | ThorchainSaversWithdrawQuoteResponseSuccess
   | ThorchainSaversCommonQuoteResponseError
+
+export type ThorchainRunepoolInformationsResponseSuccess = {
```

Comment:
nitpick (same with `informations` occurences across the PR)
```suggestion
export type ThorchainRunepoolInformationResponseSuccess = {
```

Reply:
Fixed in [cb8c726](https://github.com/shapeshift/web/pull/7436/commits/cb8c726c4618f4cef890ee41b1dbd3c95e1fe658)